soyjs/exec_test.go

```diff
@@ -777,39 +1011,63 @@ func runExecTests(t *testing.T, tests []execTest) {
 			test.output,
 			test.data,
 			test.ok,
+			nil,
 		})
 	}
 	runNsExecTests(t, nstest)
 }
 
+var pluralFuncTmpl = `
+	var soy = soy || {};
+	soy.$$pluralIndex = function(n){
+		%s
+	};
+`
+
 func runNsExecTests(t *testing.T, tests []nsExecTest) {
 	var js = initJs(t)
+
+TESTS_LOOP:
 	for _, test := range tests {
 		var js = js.Copy()
 
 		// Parse the templates, generate and run the compiled javascript.
 		var source bytes.Buffer
 		for _, input := range test.input {
+			if test.msgs != nil && test.msgs.locale != "" {
+				fbody, _ := pluralFuncBodies[test.msgs.locale]
```

Comment:
I didn't think it was worth being that defensive in a test.  I can add it though if you would prefer it (fail the test?)